app/lib/requires_make_decisions_permission.rb

```diff
@@ -0,0 +1,17 @@
+module RequiresMakeDecisionsPermission
```

Comment:
I'd like to avoid `includes` / concerns where possible, since they make it harder to find the place where a method is defined. Can we move this into `ProviderInterfaceController` instead?

Reply:
For what it's worth, this is the nightmare scenario I'm thinking about when dealing with includes:

https://github.com/alphagov/whitehall/blob/98287c46f7ee7802759e8df123ca1e06519a92dc/app/models/edition.rb#L4-L35

https://github.com/alphagov/whitehall/blob/98287c46f7ee7802759e8df123ca1e06519a92dc/app/models/case_study.rb#L2-L10

Reply:
Nice